.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,10 +5,14 @@
 ## Description
 <!-- Please describe what you have changed or added -->
 
-## How has this been tested?
+## Testing Instructions
 <!-- Please describe in detail how you tested your changes. -->
 <!-- Include details of your testing environment, tests ran to see how -->
 <!-- your change affects other areas of the code, etc. -->
```

Comment:
I think we could also delete this text in favour of the new text, which is much better in my opinion. What do you think?

Reply:
Done.